homeassistant/generated/config_flows_v2.json

```diff
@@ -0,0 +1,1417 @@
+    },
+    "adax": {
+      "name": "Adax"
+    },
+    "adguard": {
+      "name": "AdGuard Home"
+    },
+    "advantage_air": {
+      "name": "Advantage Air"
+    },
+    "aemet": {
+      "name": "AEMET OpenData"
+    },
+    "agent_dvr": {
+      "name": "Agent DVR"
+    },
+    "airly": {
+      "name": "Airly"
+    },
+    "airnow": {
+      "name": "AirNow"
+    },
+    "airthings": {
+      "name": "Airthings"
+    },
+    "airtouch4": {
+      "name": "AirTouch 4"
+    },
+    "airvisual": {
+      "name": "AirVisual"
+    },
+    "airzone": {
+      "name": "Airzone"
+    },
+    "aladdin_connect": {
+      "name": "Aladdin Connect"
+    },
+    "alarmdecoder": {
+      "name": "AlarmDecoder"
+    },
+    "almond": {
+      "name": "Almond"
+    },
+    "amberelectric": {
+      "name": "Amber Electric"
+    },
+    "ambiclimate": {
+      "name": "Ambiclimate"
+    },
+    "ambient_station": {
+      "name": "Ambient Weather Station"
+    },
+    "android_ip_webcam": {
+      "name": "Android IP Webcam"
+    },
+    "androidtv": {
+      "name": "Android TV"
+    },
+    "anthemav": {
+      "name": "Anthem A/V Receivers"
+    },
+    "apple_tv": {
+      "name": "Apple TV"
+    },
+    "arcam_fmj": {
+      "name": "Arcam FMJ Receivers"
+    },
+    "aseko_pool_live": {
+      "name": "Aseko Pool Live"
+    },
+    "asuswrt": {
+      "name": "ASUSWRT"
+    },
+    "atag": {
+      "name": "Atag"
+    },
+    "august": {
+      "name": "August"
+    },
+    "aurora": {
+      "name": "Aurora",
+      "translated_name": true
+    },
+    "aurora_abb_powerone": {
+      "name": "Aurora ABB PowerOne Solar PV"
+    },
+    "aussie_broadband": {
+      "name": "Aussie Broadband"
+    },
+    "awair": {
+      "name": "Awair"
+    },
+    "axis": {
+      "name": "Axis"
+    },
+    "azure_devops": {
+      "name": "Azure DevOps"
+    },
+    "azure_event_hub": {
+      "name": "Azure Event Hub"
+    },
+    "baf": {
+      "name": "Big Ass Fans"
+    },
+    "balboa": {
+      "name": "Balboa Spa Client"
+    },
+    "blebox": {
+      "name": "BleBox devices"
+    },
+    "blink": {
+      "name": "Blink"
+    },
+    "bluemaestro": {
+      "name": "BlueMaestro"
+    },
+    "bluetooth": {
+      "name": "Bluetooth"
+    },
+    "bmw_connected_drive": {
+      "name": "BMW Connected Drive"
+    },
+    "bond": {
+      "name": "Bond"
+    },
+    "bosch_shc": {
+      "name": "Bosch SHC"
+    },
+    "braviatv": {
+      "name": "Sony Bravia TV"
+    },
+    "broadlink": {
+      "name": "Broadlink"
+    },
+    "brother": {
+      "name": "Brother Printer"
+    },
+    "brunt": {
+      "name": "Brunt Blind Engine"
+    },
+    "bsblan": {
+      "name": "BSB-Lan"
+    },
+    "bthome": {
+      "name": "BTHome"
+    },
+    "buienradar": {
+      "name": "Buienradar"
+    },
+    "canary": {
+      "name": "Canary"
+    },
+    "cast": {
+      "name": "Google Cast"
+    },
+    "cert_expiry": {
+      "name": "Certificate Expiry",
```

Comment:
We don't need a name if we should translate it. We can just omit this. Maybe we should even just have a list stored at top-level of this file with integrations that need translated name. And then we can remove `translated_name` too.

Reply:
Fixed